using-kubecost/navigating-the-kubecost-ui/saved-reports/cost-center-report.md

```diff
@@ -1,10 +1,14 @@
 # Cost Center Report
 
 {% hint style="warning" %}
-Cost Center Report is a beta feature. Read the documentation carefully.
+Cost Center Report is a beta feature. Please share your feedback as we are in active development of this feature.
```

Comment:
The original text is how we describe all beta features and I'd like to leave that as is. "Beta" implies the feature is in active development and is receptive to user feedback, correct? Want to get @kwombach12's feedback here

Reply:
I didn't see your comment until just now. Let me know and I can re submit.

Reply:
No worries, I was out for a few days and didn't see this. The rest of the changes look good so we can roll this out while we wait for a response.